frontend/src/pages/NewRun.test.tsx

```diff
@@ -33,6 +33,10 @@ class TestNewRun extends NewRun {
   public async _pipelineSelectorClosed(confirmed: boolean): Promise<void> {
     return await super._pipelineSelectorClosed(confirmed);
   }
+
+  public _updateRecurringRunState(isRecurringRun: boolean): void {
```

Comment:
Wondering if you can just do something like "public x = super.x;" rather than a function wrapper.

Reply:
Yep! Good call